internal/events/kafka.go

```diff
@@ -0,0 +1,115 @@
+package events
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/segmentio/kafka-go"
+	"github.com/stellar/go/support/log"
+	"golang.org/x/exp/maps"
+)
+
+type KafkaEventManager struct {
+	handlers []EventHandler
+	writer   *kafka.Writer
+	reader   *kafka.Reader
+}
+
+func NewKafkaEventManager(brokers []string, consumerTopics []string, consumerGroupID string) (*KafkaEventManager, error) {
+	k := KafkaEventManager{}
+
+	writer := kafka.NewWriter(kafka.WriterConfig{
```

Comment:
`kafka.newWriter` and `kafka.WriterConfig` seem to be deprecated. I think the following should work instead: 
```
writer := &kafka.Writer{
  Addr:     kafka.TCP(brokers...),
  Balancer: &kafka.RoundRobin{},
  RequiredAcks: -1,
}
```